aws-blog-athena-importing-hive-metastores/exportdatabase.py

```diff
@@ -0,0 +1,40 @@
+import sys, os
+
+schema=sys.argv[1]
+
+command="hive --database "+schema+" -e 'show tables' -S"
+tables=[]
+separator='!echo \;;'
+
+os.system("rm "+schema+"*.hql 2> /dev/null")
+os.system("rm "+schema+"output 2> /dev/null")
+
+p = os.popen(command,"r")
+while 1:
```

Comment:
there's a cleaner way to do this without the risk of the infinite loop. 

sorry if syntax is a little off but you will see the idea

while line = p.readline():
     tables.append(line)

OR

line = p.readline()
while line:
    append
    line = p.readline()